packages/cli/__tests__/zosfiles/__system__/delete/ds/__snapshots__/cli.files.delete.ds.system.test.ts.snap

```diff
@@ -35,10 +32,7 @@ exports[`Delete Data Set Success scenarios should delete a partitioned data set
   \\"message\\": \\"\\",
   \\"stdout\\": \\"Data set deleted successfully.\\\\n\\",
   \\"stderr\\": \\"\\",
-  \\"data\\": {
-    \\"success\\": true,
-    \\"commandResponse\\": \\"Data set deleted successfully.\\"
```

Comment:
is this possibly reflecting a breaking change? 

If someone deletes a dataset with `--rfj`, should they still get the `data.commandResponse` and `data.success` values back?

Reply:
is it necessary to keep this in the test? I just removed  it because it didn't seem necessary since im also checking for an error code @zFernand0

Reply:
After adding `commandParameters.response.data.setObj(response);` back, you may need to run system tests again.
And this `data: { }` object should be filled again 😋